docs/vcs.rst

```diff
@@ -1,17 +1,20 @@
 VCS Integration
 ===============
 
+If you want to integrate editing into your own theme, you will have to declare few variables inside your configuration file (config.py) in the "html_context" object, for the template to use them. 
```

Comment:
"html_context" object -> \`\`'html_context'\`\` setting

We could also add a link to http://www.sphinx-doc.org/en/1.5.2/config.html#confval-html_context